internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -470,6 +470,13 @@ type Deployment struct {
 	Ownership *Ownership `gorm:"polymorphic:Entity;"`
 }
 
+func (d *Deployment) BeforeUpdate(tx *gorm.DB) (err error) {
+	if tx.Statement.Changed() { // if any fields changed
+		tx.Statement.SetColumn("UpdatedAt", time.Now())
```

Comment:
wasn't this working already at the db it self before ?

Reply:
nop, we are updating this manually in update state

Reply:
ok, so we probably need to add it to other tables as well or add a trigger in db
anyhow resolving this